easybuild/easyblocks/generic/intelbase.py

```diff
@@ -285,6 +294,8 @@ def install_step(self, silent_cfg_names_map=None, silent_cfg_extras=None):
             'activation': self.cfg['license_activation'],
             'license_file': self.license_file,
             'install_dir': silent_cfg_names_map.get('install_dir', self.installdir),
+            'install_mode': silent_cfg_names_map.get('install_mode', INSTALL_MODE),
+            'install_mode_name': silent_cfg_names_map.get('install_mode_name', INSTALL_MODE_NAME),
```

Comment:
this is a backwards incompatible changes, since you're defaulting to the new name here

This implies that all other easyblocks deriving from have to be updated accordingly...